ui/src/app/services/server-config.service.ts

```diff
@@ -0,0 +1,24 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {ReplaySubject} from 'rxjs/ReplaySubject';
+
+import {ErrorHandlingService} from 'app/services/error-handling.service';
+
+import {ConfigResponse, ConfigService} from 'generated';
+
+@Injectable()
+export class ServerConfigService {
+  private configObs: Observable<ConfigResponse>;
+  constructor(
+    private configService: ConfigService, private errorHandlingService: ErrorHandlingService) {}
```

Comment:
Nit: everywhere else we've been doing 
```
constructor(
  dep: SomeDependency,
  /* etc */
) {}
```
(or sometimes a vertically aligned indent like in Python) for constructors with too many deps for a single line.

Reply:
Done.